taca/log/__init__.py

```diff
@@ -19,16 +20,16 @@
 }
 
 def init_logger_file(log_file, log_level='INFO'):
-    """ Append a FileHandler to the general logger.
+    """ Append a FileHandler to the root logger.
 
     :param str log_file: Path to the log file
     :param str log_level: Logging level
     """
     log_level = LOG_LEVELS[log_level] if log_level in LOG_LEVELS.keys() else logging.INFO
 
-    LOG.setLevel(log_level)
+    ROOT_LOG.setLevel(log_level)
```

Comment:
Actually here the log-level is updated from the hardcoded level set above, no? So everything printed to the file handler will also go to the stream handler? I'm confused...